<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="rectangleBtn">
     <property name="toolTip">
      <string>画矩形</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/icons/ScreenShot/control_01.jpg</normaloff>:/icons/ScreenShot/control_01.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="drawRoundBtn">
     <property name="toolTip">
      <string>画圆</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/icons/ScreenShot/control_02.jpg</normaloff>:/icons/ScreenShot/control_02.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="arrowBtn">
     <property name="toolTip">
      <string>画箭头</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/icons/ScreenShot/control_03.jpg</normaloff>:/icons/ScreenShot/control_03.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="drawLineBtn">
     <property name="toolTip">
      <string>画笔</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/icons/ScreenShot/control_04.jpg</normaloff>:/icons/ScreenShot/control_04.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="mosaicBtn">
     <property name="toolTip">
      <string>马赛克</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/icons/ScreenShot/control_05.jpg</normaloff>:/icons/ScreenShot/control_05.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="textEditBtn">
     <property name="toolTip">
      <string>添加文字</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/icons/ScreenShot/control_06.jpg</normaloff>:/icons/ScreenShot/control_06.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="moveCursorBtn">
     <property name="toolTip">
      <string>恢复指针</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="returneditBtn">
     <property name="toolTip">
      <string>撤销操作</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/icons/ScreenShot/control_07.jpg</normaloff>:/icons/ScreenShot/control_07.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="saveBtn">
     <property name="toolTip">
      <string>保存截图</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/icons/ScreenShot/control_08.jpg</normaloff>:/icons/ScreenShot/control_08.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="ocrBtn">
     <property name="toolTip">
      <string>图片转文字</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/icons/ScreenShot/control_14.png</normaloff>:/icons/ScreenShot/control_14.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="cancelBtn">
     <property name="toolTip">
      <string>退出截图</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/icons/ScreenShot/control_12.jpg</normaloff>:/icons/ScreenShot/control_12.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="finishBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>完成截图</string>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/icons/ScreenShot/control_13.jpg</normaloff>:/icons/ScreenShot/control_13.jpg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
